packages/admin/src/targetGroups/TargetGroupsGrid.tsx

```diff
@@ -128,9 +128,9 @@ export function TargetGroupsGrid({
     const dataGridProps = { ...useDataGridRemote(), ...usePersistentColumnState("TargetGroupsGrid") };
 
     const targetGroupContactsQuery = gql`
-        query TargetGroupContacts($targetGroupId: ID, $offset: Int, $limit: Int, $scope: EmailCampaignContentScopeInput!) {
-            brevoContacts(targetGroupId: $targetGroupId, offset: $offset, limit: $limit, scope: $scope) {
-                nodes {
+        query TargetGroupContacts($targetGroupId: ID!, $offset: Int, $limit: Int, $scope: EmailCampaignContentScopeInput!) {
+            brevoContacts(targetGroupId: $targetGroupId, offset: $offset, limit: $limit, scope: $scope) {               
+                 nodes {
```

Comment:
Out of scope: Shouldn't this be formatted and linted so that these unnecessary whitespace changes don't happen?

Reply:
it formats the whitespaces automatically when saving, and in the pipeline it fails when having wrong whitespaces: https://github.com/vivid-planet/comet-brevo-module/pull/54

The only thing I have just seen is that the husky precommit doesn't work correctly. I'll look into it.